content/shmem_scan.tex

```diff
@@ -0,0 +1,121 @@
+\apisummary {
+  Performs inclusive or exclusive prefix sum operations
+}
+
+\begin{apidefinition}
+
+%% C11
+\begin{C11synopsis}
+int @\FuncDecl{shmem\_sum\_inscan}@(shmem_team_t team, TYPE *dest, const TYPE *source, size_t nreduce);
+int @\FuncDecl{shmem\_sum\_exscan}@(shmem_team_t team, TYPE *dest, const TYPE *source, size_t nreduce);
+\end{C11synopsis}
+where \TYPE{} is one of the integer, real, or complex types supported
+for the SUM operation as specified by Table \ref{teamreducetypes}.
+
+%% C/C++
+\begin{Csynopsis}
+int @\FuncDecl{shmem\_\FuncParam{TYPENAME}\_sum\_inscan}@(shmem_team_t team, TYPE *dest, const TYPE *source, size_t nreduce);
+int @\FuncDecl{shmem\_\FuncParam{TYPENAME}\_sum\_exscan}@(shmem_team_t team, TYPE *dest, const TYPE *source, size_t nreduce);
+\end{Csynopsis}
+where \TYPE{} is one of the integer, real, or complex types supported
+for the SUM operation and has a corresponding \TYPENAME{} as specified
```

Comment:
Can we include a reference to the reduction table, and also update the caption for the table to be inclusive of scan operations?

Reply:
> Can we include a reference to the reduction table

This next line references the team-based reduction table of types, names, and operations. Is there some other table you want referenced?

> and also update the caption for the table to be inclusive of scan operations?

I'm hesitant to caption the table "~Reduction~ Types, Names, and Supporting Operations for Team-Based Reductions **and Scans"** because the scan support only applies to the SUM operation. Maybe that's okay, but it feels to me like it adds ambiguity not reduces it. Would such a caption seem to imply scan supports more than just SUM operations?